addons/mail/models/mail_channel.py

```diff
@@ -366,46 +364,20 @@ def _notify_compute_recipients(self, message, msg_vals):
         """
         # get values from msg_vals or from message if msg_vals doen't exists
         msg_sudo = message.sudo()
-        mailing_channels = self.filtered('email_send')
         message_type = msg_vals.get('message_type', 'email') if msg_vals else msg_sudo.message_type
         pids = msg_vals.get('partner_ids', []) if msg_vals else msg_sudo.partner_ids.ids
 
         # notify only user input (comment or incoming emails)
         if message_type not in ('comment', 'email'):
             return []
         # notify only mailing lists or if mentioning recipients
-        if not mailing_channels and not pids:
+        if not pids:
             return []
 
         email_from = tools.email_normalize(msg_vals.get('email_from') or msg_sudo.email_from)
         author_id = msg_vals.get('author_id') or msg_sudo.author_id.id
 
         recipients_data = []
-        if mailing_channels:
```

Comment:
Note that in ``_notify_email_recipient_values`` you can remove the whole code I think. Reasons: 
1/ use of blacklist was targeting mainly "big groups" used a mailing lists. It should not work as other records, no need to keep a specific behavior.
2/ the use of ``email_to`` was also intended for "big groups" to simplify sending emails in batch. This is not really a requirement of channels anymore as this will be replaced by real mailing lists.

This could be explained in commit message also, as it is interesting to explain.